package.json

```diff
@@ -107,9 +109,10 @@
     "supertest": "^4.0.2"
   },
   "dependencies": {
-    "@babel/runtime": "^7.11.2",
```

Comment:
Losing babel/runtime is surprising to me, was that intended? It looks likely to be the cause of the deploy step not working.

Reply:
Will fix.